### ISSUE TITLE
Fedora 33 released but in repo available for 31  only

### ISSUE THREAD

**Issue by norguhtar** [tarantool/tarantool#5502]
Tarantool version: 2.4

OS version: Fedora 32 and Fedora 33

Bug description: repo not exist 

Steps to reproduce: try install repo on Fedora 32 and 33
```sh
 curl -L https://tarantool.io/ByZVhS/release/2.4/installer.sh | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   184  100   184    0     0    696      0 --:--:-- --:--:-- --:--:--   699
100  9357  100  9357    0     0  11259      0 --:--:-- --:--:-- --:--:--     0
Detected operating system as fedora/32.
Unfortunately, your operating system is not supported by this script.
```

If try manual 
```sh
dnf -y install sudo
sudo rm -f /etc/yum.repos.d/*tarantool*.repo
sudo tee /etc/yum.repos.d/tarantool_2_4.repo <<- EOF
[tarantool_2_4]
name=Fedora-\$releasever - Tarantool
baseurl=https://download.tarantool.org/tarantool/release/2.4/fedora/\$releasever/x86_64/
gpgkey=https://download.tarantool.org/tarantool/release/2.4/gpgkey
repo_gpgcheck=1
gpgcheck=0
enabled=1

[tarantool_2_4-source]
name=Fedora-\$releasever - Tarantool Sources
baseurl=https://download.tarantool.org/tarantool/release/2.4/fedora/\$releasever/SRPMS
gpgkey=https://download.tarantool.org/tarantool/release/2.4/gpgkey
repo_gpgcheck=1
gpgcheck=0
EOF

sudo dnf -q makecache -y --disablerepo='*' --enablerepo='tarantool_2_4'
sudo dnf -y install tarantool
- Status code: 404 for https://hb.bizmrg.com/tarantool_repo/release/2.4/fedora/32/x86_64/repodata/repomd.xml (IP: 95.163.144.221)
Error: Не удалось загрузить метаданные для репозитория «tarantool_2_4»: Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried


```

**Comment by avtikhon** [comment]
Resolving issues with the following commits:

```
commit 86b34b096ce18cacfd23002df18205946d06e81c
Author: Alexander V. Tikhonov <avtikhon@tarantool.org>
Date:   Tue Jan 12 17:04:14 2021 +0300

    gitlab-ci: add Fedora 33 to packaging
    
    Added jobs for testing and deploing Fedora 33 packages.
    
    Found that running Fedora 33 docker image w/o '--privileged' flag
    failed to show any of the paths:
    
      $ ls -al
      ls: cannot access '.': Operation not permitted
      ls: cannot access '..': Operation not permitted
      ls: cannot access 'build': Operation not permitted
      ...
    
      total 0
      d????????? ? ? ? ?            ? .
      d????????? ? ? ? ?            ? ..
      -????????? ? ? ? ?            ? .appveyor.yml
      -????????? ? ? ? ?            ? .editorconfig
      -????????? ? ? ? ?            ? .gdbinit
      d????????? ? ? ? ?            ? .git
    
    To fix it '--priviliged' flag was added to Fedora 33 packaging jobs
    configurations.
    
    Added Fedora 33 at update_repo tool to make able to save packages
    in S3 buckets.
    
    Closes #5502

commit 11a3cceba2200ccaf0657ad05b99a6799dd77057
Author: Alexander V. Tikhonov <avtikhon@tarantool.org>
Date:   Mon Jan 11 15:09:48 2021 +0300

    Enable packaging for Fedora 33
    
    Since Python 2 was depricated then new OS do not have repositories
    with its packages. It is not good way to create all these packages
    manually and store it as backported. Let's wait when Python 3 will
    be used for testing and for now the testing disabled for Fedora 33
    packaging.
    
    Also found that %cmake macros used in build spec was changed - new
    option was added:
    
      -B x86_64-redhat-linux-gnu
    
    It changed the build path and broke the build, to fix it these
    options was additionaly localy reset to:
    
      -B .
    
    Found that LTO was set by default in Fedora 33, check
    
      /usr/lib/rpm/redhat/macros
    
        # LTO is the default in Fedora.
        #   "%define _lto_cflags %{nil}"  to opt out
        #
        # We currently have -ffat-lto-objects turned on out of an abundance of
        # caution.  To remove it we need to do a check of the installed .o/.a files
        # to verify they have real sections/symbols after LTO stripping.  That
        # way we can detect installing an unusable .o/.a file.  This is on the TODO
        # list for F34.
        %_gcc_lto_cflags        -flto=auto -ffat-lto-objects
        %_clang_lto_cflags      -flto
        %_lto_cflags            %{expand:%%{_%{toolchain}_lto_cflags}}
    
        %_general_options       -O2 %{?_lto_cflags} -fexceptions -g -grecord-gcc-switches -pipe
    
    and build issues occured:
    
      /build/usr/src/debug/tarantool-2.8.0.2/src/lib/core/reflection.h:124:33: warning: type of â€˜METHODS_SENTINELâ€™ does not match original declaration [-Wlto-type-mismatch]
        124 | extern const struct method_info METHODS_SENTINEL;
            |                                 ^
      /build/usr/src/debug/tarantool-2.8.0.2/src/lib/core/reflection.c:35:26: note: â€˜METHODS_SENTINELâ€™ was previously declared here
         35 | const struct method_info METHODS_SENTINEL = {
            |                          ^
    
    Needed for #5502
```

Still need to resolve found issues:
https://gitlab.com/tarantool/tarantool/-/jobs/957029728
@artemreyt could you please help ?

**Comment by avtikhon** [comment]
Made investigations and left suggestions on fixes:
https://github.com/tarantool/tarantool/issues/5697#issuecomment-760067489
